frontend/src/routes/Home/Search/components/Searchbar.tsx

```diff
@@ -198,7 +199,7 @@ export function Searchbar(props: SearchbarProps) {
       )
       .map((currentValue) => (
         <MenuItem
-          isDisabled={currentValue.name.includes('Loading')}
+          isDisabled={currentValue.name.includes('Loading') || currentValue.disabled}
```

Comment:
Is the pre-existing logic here compatible with translation?

Reply:
@KevinFCormier logic should now account for translation.